src/app/clusters/software-diagnostics-server/software-diagnostics-cluster.h

```diff
@@ -38,15 +38,12 @@ namespace Clusters {
 /// Translates between matter calls and Software Diagnostics logic
 ///
 /// This cluster is expected to only ever exist on endpoint 0 as it is a singleton cluster.
-template <typename LOGIC>
-class SoftwareDiagnosticsServerCluster : public DefaultServerCluster,
-                                         public SoftwareDiagnostics::SoftwareFaultListener,
-                                         private LOGIC
+class SoftwareDiagnosticsServerCluster : public DefaultServerCluster, public SoftwareDiagnostics::SoftwareFaultListener
 {
 public:
     template <typename... Args>
```

Comment:
I may be missing something, but this `template <type name... Args>` is still needed after the rest of the changes?

Reply:
removed.